test/unit/core/simd/detail/builtin_simd_test.cpp

```diff
@@ -169,6 +186,16 @@ TEST(builtin_simd, simd_traits)
     EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<uint16x16_t>::swizzle_type, uint8x32_t>));
     EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<uint32x8_t>::swizzle_type, uint8x32_t>));
     EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<uint64x4_t>::swizzle_type, uint8x32_t>));
+
+    EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<int8x32_t>::rebind<uint8_t>, uint8x32_t>));
+    EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<int16x16_t>::rebind<uint16_t>, uint16x16_t>));
+    EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<int32x8_t>::rebind<uint32_t>, uint32x8_t>));
+    EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<int64x4_t>::rebind<uint64_t>, uint64x4_t>));
+
+    // EXPECT_TRUE((std::is_same_v<seqan3::simd::simd_traits<float>::rebind<uint32_t>, uint32x4_t>));
```

Comment:
Please add a comment, that currently simd_concept does not support floating point scalar types.